### PR TITLE
SLI-895 Add support for Go in GoLand

### PR DESCRIPTION
**Pull request by nquinquenel**

### CODE REVIEW

File: goland/src/main/java/org/sonarlint/intellij/goland/GoLanguageActivator.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+import java.util.Set;
+import org.sonarlint.intellij.common.LanguageActivator;
+import org.sonarsource.sonarlint.core.commons.Language;
+
+public class GoLanguageActivator implements LanguageActivator {
+  @Override
+  public void amendLanguages(Set<Language> enabledLanguages) {
+    // Only Go for now in GoLand
+    enabledLanguages.clear();
+    enabledLanguages.add(Language.GO);
```
Comment (damien-urruty-sonarsource) [inline goland/src/main/java/org/sonarlint/intellij/goland/GoLanguageActivator.java:31]: To check with Marco but I think we also want to enable the Secrets language

**Review by damien-urruty-sonarsource — APPROVED** [review]
One comment, else LGTM!

**Comment by damien-urruty-sonarsource** [comment]
And we probably need to find a way to test that. A new axis for UI tests ?

File: common/src/main/java/org/sonarlint/intellij/common/util/SonarLintUtils.java [hunk 2]
```diff
@@ -208,7 +208,7 @@ private static ApplicationInfo getAppInfo() {
 
   public static boolean isTaintVulnerabilitiesEnabled() {
     // No Taint Vulnerabilities in C/C++ for the time being
```
Comment (damien-urruty-sonarsource) [inline common/src/main/java/org/sonarlint/intellij/common/util/SonarLintUtils.java:210]: Maybe remove this comment that is now outdated ? Or update it ?

File: goland/src/main/java/org/sonarlint/intellij/goland/GoLanguageActivator.java [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+package org.sonarlint.intellij.goland;
+
+import java.util.Set;
+import org.sonarlint.intellij.common.LanguageActivator;
+import org.sonarsource.sonarlint.core.commons.Language;
+
+public class GoLanguageActivator implements LanguageActivator {
+  @Override
+  public void amendLanguages(Set<Language> enabledLanguages) {
+    // Only Go for now in GoLand
```
Comment (damien-urruty-sonarsource) [inline goland/src/main/java/org/sonarlint/intellij/goland/GoLanguageActivator.java:29]: Same here, Secrets are also supported

**Review by damien-urruty-sonarsource — APPROVED** [review]

File: goland/src/main/java/org/sonarlint/intellij/goland/GoLanguageActivator.java (hunk 3 above)
Comment (nquinquenel) [reply goland/src/main/java/org/sonarlint/intellij/goland/GoLanguageActivator.java:29]: I think it's pretty self explanatory, I removed them

**Pull request merged**